src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -665,6 +665,7 @@ public function formAction($id, Request $request)
             'drawerModules' => $drawerModules,
             'layoutTitle' => $this->trans('Product', 'Admin.Global'),
             'isProductPageV2Enabled' => ($this->isProductPageV2Enabled()),
+            'isCreationMode' => (int) $product->state === Product::STATE_TEMP,
```

Comment:
```suggestion
            'isCreationMode' => $product->state === Product::STATE_TEMP,
```

I see no reason to use int when having `isSomething` as name

Reply:
I was afraid this condition might fail if `$product->state` happened to be a string `'0'` , while `Product::STATE_TEMP` is an integer `0`

Reply:
oh my bad, I miss read because of parenthesis :sweat_smile: 
I read `(int) ($product->state === Product::STATE_TEMP)`